src/views/ConversationView.tsx

```diff
@@ -8,6 +8,16 @@ import Header from "../components/Header";
 import { Cog6ToothIcon } from "@heroicons/react/24/solid";
 import { useLiveConversation } from "../hooks/useLiveConversation";
 import ConversationSettingsView from "./ConversationSettingsView";
+import { ContentTypeReaction } from "../model/reactions";
+import { ContentTypeId } from "@xmtp/xmtp-js";
+
+function appearsInMessageList(message: Message): boolean {
```

Comment:
i know you're not a fan of arrow funcs, but this would be a great use case for one.